libraries/bezier-rs/src/bezier/core.rs

```diff
@@ -141,13 +142,34 @@ impl Bezier {
 		}
 	}
 
-	/// Convert `Bezier` to SVG `path`.
-	pub fn to_svg(&self) -> String {
+	/// Returns an SVG representation of the `Bezier`.
+	pub fn to_svg(&self, options: ToSVGOptions) -> String {
+		let anchor_arguments = options.formatted_anchor_arguments();
+		let anchor_circles = format!(
+			r#"<circle cx="{}" cy="{}" {}/><circle cx="{}" cy="{}" {}/>"#,
+			self.start.x, self.start.y, anchor_arguments, self.end.x, self.end.y, anchor_arguments
+		);
+
+		let handle_point_arguments = options.formatted_handle_point_arguments();
+		let handle_circles = match self.handles {
+			BezierHandles::Linear => None,
+			BezierHandles::Quadratic { handle } => Some(format!(r#"<circle cx="{}" cy="{}" {}/>"#, handle.x, handle.y, handle_point_arguments)),
+			BezierHandles::Cubic { handle_start, handle_end } => Some(format!(
+				r#"<circle cx="{}" cy="{}" {}/><circle cx="{}" cy="{}" {}/>"#,
+				handle_start.x, handle_start.y, handle_point_arguments, handle_end.x, handle_end.y, handle_point_arguments
```

Comment:
Here and elsewhere, when possible just stick variables directly into the `{}`, I think it makes it more readable. So in this case, handle_point_arguments could go in the string directly.

Reply:
I think I got all the locations that can be changed, but I would appreciate another quick look!